TROUBLESHOOTING.md

```diff
@@ -87,18 +87,20 @@ If you think you're hitting an out-of-memory situation, you can alleviate this p
 
 Pylance (by default) runs using VS Code's Node.js executable (which has the 4GB limit). 
 
-To specify your own Node.js executable, set this setting in your User settings.json and restart VS Code:
+To specify your own Node.js executable, set this setting in your User settings.json or Remote settings.json and restart VS Code:
 
 ```json
 "python.analysis.nodeExecutable": "<path to node.js exe>"
 ```
 
+* If you set `auto` as `the path to the Node.js executable`, pylance will automatically download the appropriate version from [Node](https://nodejs.org/dist/) and use it without requiring manual installation. This should also work in remote environments.
```

Comment:
```suggestion
* If you set `auto` as `the path to the Node.js executable`, pylance will automatically download the appropriate version from [Node](https://nodejs.org/dist/) and use it without requiring manual installation. This will also work in remote environments.
```